11_js-basics/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -0,0 +1,101 @@
+const operand2 = 4;
+
+/*
+Mathematical operations
+*/
+const addButton = document.querySelector('[data-js="add"]');
+const subtractButton = document.querySelector('[data-js="subtract"]');
+const multiplyButton = document.querySelector('[data-js="multiply"]');
+const divideButton = document.querySelector('[data-js="divide"]');
+const exponentButton = document.querySelector('[data-js="exponent"]');
+const moduloButton = document.querySelector('[data-js="modulo"]');
+
+addButton.addEventListener("click", () => {
+  const operandAnswer = operand1 + operand2;
+  console.log("Answer 1: ", operandAnswer);
+  // 1. Add the two operands and store the result in a variable. Log the variable's value to the console.
+});
+
+subtractButton.addEventListener("click", () => {
+  const operandAnswer = operand1 - operand2;
+  console.log("Answer 2: ", operandAnswer);
+  // 2. Subtract the two operands and store the result in a variable. Log the variable's value to the console.
+});
+
+multiplyButton.addEventListener("click", () => {
+  const operandAnswer = operand1 * operand2;
+  console.log("Answer 3: ", operandAnswer);
+  // 3. Multiply the two operands and store the result in a variable. Log the variable's value to the console.
+});
+
+divideButton.addEventListener("click", () => {
+  const operandAnswer = operand1 / operand2;
+  console.log("Answer 4: ", operandAnswer);
+  // 4. Divide the two operands and store the result in a variable. Log the variable's value to the console.
+});
+
+exponentButton.addEventListener("click", () => {
+  const operandAnswer = operand1 ** operand2;
+  console.log("Answer 5: ", operandAnswer);
+  // 5. Raise "operand1" to the value of "operand2" and store the result in a variable. Log the variable's value to the console.
+});
+
+moduloButton.addEventListener("click", () => {
+  const operandAnswer = operand1 % operand2;
+  console.log("Answer 6: ", operandAnswer);
+  // 6. Calculate the remainder when dividing "operand1" by "operand2" and store the result in a variable. Log the variable's value to the console.
+});
+
+/*
+Update the first operand
+Please don't forget to think about the declaration of the variable "operand1".
+*/
+const increaseByOneButton = document.querySelector(
+  '[data-js="increase-by-one"]'
+);
+const increaseByFiveButton = document.querySelector(
+  '[data-js="increase-by-five"]'
+);
+const decreaseByOneButton = document.querySelector(
+  '[data-js="decrease-by-one"]'
+);
+const decreaseByFiveButton = document.querySelector(
+  '[data-js="decrease-by-five"]'
+);
+const multiplyByTwoButton = document.querySelector(
+  '[data-js="multiply-by-two"]'
+);
+const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
+
+increaseByOneButton.addEventListener("click", () => {
+  console.log(++operand1);
+  // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+});
+
+increaseByFiveButton.addEventListener("click", () => {
+  console.log((operand1 += 5));
```

Comment:
As far as I know, you dont need to put the math-operation into additional brackets. Works without